delta/plugins/blazegraph/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/model/BlazegraphViewsSerializationSuite.scala

```diff
@@ -48,31 +48,32 @@ class BlazegraphViewsSerializationSuite extends SerializationSuite {
 
   private val defaultIndexingValue  = IndexingBlazegraphViewValue()
   private val defaultIndexingSource = defaultIndexingValue.toJson(indexingId)
-
+  
   // format: off
-  private val blazegraphViewsMapping = VectorMap(
-    BlazegraphViewCreated(indexingId, projectRef, uuid, defaultIndexingValue, defaultIndexingSource, 1, instant, subject) ->
-      loadEvents("blazegraph", "default-indexing-view-created.json"),
-    BlazegraphViewCreated(indexingId, projectRef, uuid, indexingValue, indexingSource, 1, instant, subject)       ->
-      loadEvents("blazegraph", "indexing-view-created.json"),
-    BlazegraphViewCreated(aggregateId, projectRef, uuid, aggregateValue, aggregateSource, 1, instant, subject)    ->
-      loadEvents("blazegraph", "aggregate-view-created.json"),
-    BlazegraphViewUpdated(indexingId, projectRef, uuid, defaultIndexingValue, defaultIndexingSource, 2, instant, subject) ->
-      loadEvents("blazegraph", "default-indexing-view-updated.json"),
-    BlazegraphViewUpdated(indexingId, projectRef, uuid, indexingValue, indexingSource, 2, instant, subject)       ->
-      loadEvents("blazegraph", "indexing-view-updated.json"),
-    BlazegraphViewUpdated(aggregateId, projectRef, uuid, aggregateValue, aggregateSource, 2, instant, subject)    ->
-      loadEvents("blazegraph", "aggregate-view-updated.json"),
-    BlazegraphViewTagAdded(indexingId, projectRef, BlazegraphType, uuid, targetRev = 1, tag, 3, instant, subject) ->
-      loadEvents("blazegraph", "view-tag-added.json"),
-    BlazegraphViewDeprecated(indexingId, projectRef, BlazegraphType, uuid, 4, instant, subject)                   ->
-      loadEvents("blazegraph", "view-deprecated.json")
-  )
+  private val created = BlazegraphViewCreated(indexingId, projectRef, uuid, defaultIndexingValue, defaultIndexingSource, 1, instant, subject)
+  private val created1= BlazegraphViewCreated(indexingId, projectRef, uuid, indexingValue, indexingSource, 1, instant, subject)
+  private val created2 = BlazegraphViewCreated(aggregateId, projectRef, uuid, aggregateValue, aggregateSource, 1, instant, subject)
+  private val updated = BlazegraphViewUpdated(indexingId, projectRef, uuid, defaultIndexingValue, defaultIndexingSource, 2, instant, subject)
+  private val updated1= BlazegraphViewUpdated(indexingId, projectRef, uuid, indexingValue, indexingSource, 2, instant, subject)
```

Comment:
It seems that scalafmt needs to be run

Reply:
I turned off formatting here because scalafmt makes this part a bit unreadable in my opinion. However I just fixed it a bit manually. Let me know, if you prefer I can turn on the formatting.